site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/index.js

```diff
@@ -46,7 +46,7 @@ const TombstoneMetadata = ({ marbleItem }) => {
     },
   }
   const hasUri = typy(uriValue, '[0].value').safeArray.length > 0
-  const hasCreator = typy(creators, '[0].value').safeArray.length > 0
+  const hasCreator = (typy(creators, '[0].value').safeArray.length > 0 && (typy(creators, '[0].value').safeArray[0] !== 'unknown'))
```

Comment:
i don't think this is the correct algorithm we want. 
What we want is.  If there is a creator and it is exactly the string unknown.  Then we want it to be "Unknown Creator" 
The thing we don't want is the words unknown appearing like they are now when there is no creator and it does not make sense.  So we are moving the adding of the creator of unknow to the pipeline.